cmd/nightfalldlp/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -l
```

Comment:
A downside of the script is that it is currently very Github Actions specific

Reply:
yup. It's not a big deal now since it's pretty simple, but another downside is the fact that most of the team isn't super fluent in bash. Is it worth making a ticket to migrate this script to go? Or should we wait until it becomes a problem/we need to update this file?

Reply:
Not sure how difficult it is to migrate it to go, but I think it is worthwhile. Right now the docker image is Github Action specific so if we move it into the go code then the image can be used across all environments that we support.

Created a ticket for it